interface/src/app/gene/network/network.tpl.html

```diff
@@ -0,0 +1,15 @@
+<h1>ADAGE: Gene-Gene Network</h1>
+
+<div ng-hide="ctrl.statusMessage">
```

Comment:
So if statusMessage is not undefined or not false then this will show? What if the statusMessage is a blank string?